docs/build/build-macOS.md

```diff
@@ -14,7 +14,7 @@ to `brew` command.
 brew install --HEAD srt
 ```
 
-Install [CMake](https://cmake.org/) and OpenSSL with development files from `brew`.
+Install [CMake](https://cmake.org/) and OpenSSL with development files from `brew`. OpenSSL is better to be installed as latest version from the `brew` system rather than relying on the version that is presently installed in the system.
```

Comment:
It is recommended to install the latest version of OpenSSL from the `brew` system rather than relying on the version that is presently installed in the system.